mypy/checkexpr.py

```diff
@@ -1628,8 +1628,20 @@ def is_valid_var_arg(self, typ: Type) -> bool:
 
     def is_valid_keyword_var_arg(self, typ: Type) -> bool:
         """Is a type valid as a **kwargs argument?"""
-        return is_subtype(typ, self.chk.named_generic_type(
-            'builtins.dict', [self.named_type('builtins.str'), AnyType()]))
+        if self.chk.options.python_version[0] >= 3:
+            return is_subtype(typ, self.chk.named_generic_type(
+                'builtins.dict', [self.named_type('builtins.str'),
+                                  AnyType()]))
+        else:
+            return (
+                is_subtype(typ, self.chk.named_generic_type(
+                    'builtins.dict',
+                    [self.named_type('builtins.str'),
+                     AnyType()])) or
```

Comment:
I'd prefer not to hide this `or` behind such a long call. Maybe on a line by itself (if flake8 is okay with that) or at the start of the next line (which is now acceptable style in PEP 8)?